ui-components/src/locales/general.json

```diff
@@ -1493,18 +1493,48 @@
       "sqFeetMin": "Minimum Square Footage",
       "sqFeetMax": "Max Square Footage",
       "availability": "Availability",
-      "count": "Total Count",
+      "count": "Afforable Unit Group Quantity",
       "available": "Total Available",
       "delete": "Delete this Summary",
-      "deleteConf": "Do you really want to delete this summary?"
+      "deleteConf": "Do you really want to delete this summary?",
+      "bathroomsMin": "Min Number of Bathrooms",
+      "bathroomsMax": "Max Number of Bathrooms",
+      "openWaitlist": "Waitlist Status",
+      "open": "Open",
+      "closed": "Closed",
+      "addAmi": "Add AMI level",
+      "amiChart" :"AMI Chart",
+      "percentageOfAmi" :"Percentage of AMI",
+      "monthlyRentDeterminationType" :"How is rent determined?",
+      "flatRentValue" :"Monthly Rent",
+      "percentageOfIncomeValue" :"Percentage of Income",
+      "flatRent": "Flat Rent",
+      "percentIncome": "% of Income",
+      "amiLevel": "AMI Level",
+      "rentType": "Rent Type",
+      "deleteAmi": "Delete this AMI Level",
+      "deleteAmiConf": "Do you really want to delete this AMI Level?",
+      "studio": "Studio",
+      "oneBdrm": "1 Bedroom",
```

Comment:
We can change this later imo but we don't want these strings, we want to use the 1BR / 2BR eventually